README.md

```diff
@@ -25,8 +25,15 @@ To get a local copy up and running follow these simple example steps.
 
 ### Setup
 
-- Clone the repository on your local machine
-- cd into the folder
+- git clone https://github.com/taiwocoker/Weather-App.git
+- cd WeatherApp
+- Copy the absolute path of the index file, paste on your browser and click on enter
+
+### for webpack run:
+
+$ npm run build
+
+$ npm run start
```

Comment:
Opps, 🙂 , I used the previous screenshot as an example; you don't have the scripts `start` in your `package.json`
Just try to write the steps that take you to run your app locally.

**I trust you can update the README before merging.**